spanner/changestreams/README.md

```diff
@@ -0,0 +1,48 @@
+# Cloud Spanner Change Stream Sample
+
+## Setup
+
+This sample requires [Java](https://www.java.com/en/download/) and [Maven](http://maven.apache.org/) for building the application.
+
+1.  **Follow the set-up instructions in [the documentation](https://cloud.google.com/java/docs/setup).**
+
+2.  Enable APIs for your project.
+    [Click here](https://console.cloud.google.com/flows/enableapi?apiid=spanner.googleapis.com&showconfirmation=true)
+    to visit Cloud Platform Console and enable the Google Cloud Spanner API.
+
+3.  Create a Cloud Spanner instance and database via the Cloud Plaform Console's
+    [Cloud Spanner section](http://console.cloud.google.com/spanner).
+
+4.  Enable application default credentials by running the command `gcloud auth application-default login`.
+
+## Build and Run
```

Comment:
Is there anyway we could make the sample self-contained, meaning that in a single execution it creates a table / change streams, inserts data, queries the change stream, prints out the records and finally drop the table and change stream?

Reply:
They were all together in the original PR: https://github.com/googleapis/java-spanner/pull/1565, I separated them as per our discussion which was long time ago. I don't mind either way, I actually think do them all together is better - let me know your final thoughts, I can change accordingly

Reply:
Ah ok, thanks for reminding me. I think that we separated them so that it would be easier to show them in separate sample sections. We could also tag sections within the same file if you feel that is better.

Reply:
Sorry by tagging sections you meant this? https://idratherbewriting.com/java-javadoc-tags/

Reply:
Oh sorry, I should had explained it better.

We use comments to denote when a sample starts / ends in the Java file. This is what I mean by tags. These are arbitrary tags, which are given to the docs team so that they know when a sample starts and ends. See an example here: [start tag](https://github.com/googleapis/java-spanner/blob/main/samples/snippets/src/main/java/com/example/spanner/QueryWithJsonParameterSample.java#L19), [end tag](https://github.com/googleapis/java-spanner/blob/main/samples/snippets/src/main/java/com/example/spanner/QueryWithJsonParameterSample.java#L64)

Reply:
Thank you! Done